homeworks/rostyslav.havuka_RostyslavHavuka/6-typeScript/src/main.css

```diff
@@ -0,0 +1,54 @@
+.red-square {
+  background-color: rgb(255, 0, 0);
```

Comment:
Some properties use hex `#xxx`, some properties use `rgb(xxx, xxx, xxx)`.
Please, find the one style.

I suggest you use hex everywhere